src/main/java/it/gov/pagopa/payhub/activities/service/ingestionflow/receipt/ReceiptProcessingService.java

```diff
@@ -0,0 +1,94 @@
+import it.gov.pagopa.payhub.activities.connector.debtposition.ReceiptService;
+import it.gov.pagopa.payhub.activities.dto.ingestion.receipt.ReceiptErrorDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.receipt.ReceiptIngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.receipt.ReceiptIngestionFlowFileResult;
+import it.gov.pagopa.payhub.activities.mapper.ingestionflow.receipt.ReceiptMapper;
+import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowProcessingService;
+import it.gov.pagopa.pu.debtposition.dto.generated.ReceiptWithAdditionalNodeDataDTO;
+import it.gov.pagopa.pu.processexecutions.dto.generated.IngestionFlowFile;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+@Service
+@Lazy
+@Slf4j
+public class ReceiptProcessingService extends IngestionFlowProcessingService<ReceiptIngestionFlowFileDTO, ReceiptIngestionFlowFileResult, ReceiptErrorDTO> {
+
+    private final ReceiptService receiptService;
+    private final ReceiptMapper receiptMapper;
+
+    public ReceiptProcessingService(ReceiptMapper receiptMapper,
+                                    ReceiptErrorsArchiverService receiptErrorsArchiverService,
+                                    ReceiptService receiptService) {
+        super(receiptErrorsArchiverService);
+        this.receiptService = receiptService;
+        this.receiptMapper = receiptMapper;
+    }
+
+    /**
+     * Processes a stream of InstallmentIngestionFlowFileDTO and synchronizes each installment.
+     *
+     * @param iterator          Stream of installment ingestion flow file DTOs to be processed.
+     * @param readerExceptions  A list which will collect the exceptions thrown during iterator processing
+     * @param ingestionFlowFile Metadata of the ingestion file containing details about the ingestion process.
+     * @param workingDirectory  The directory where error files will be written if processing fails.
+     * @return An {@link ReceiptIngestionFlowFileResult} containing details about the processed rows, errors, and archived files.
+     */
+    public ReceiptIngestionFlowFileResult processReceipts(Iterator<ReceiptIngestionFlowFileDTO> iterator,
+                                                          List<CsvException> readerExceptions,
+                                                          IngestionFlowFile ingestionFlowFile,
+                                                          Path workingDirectory) {
+        List<ReceiptErrorDTO> errorList = new ArrayList<>();
+        ReceiptIngestionFlowFileResult result = new ReceiptIngestionFlowFileResult();
+        process(iterator, readerExceptions, result, ingestionFlowFile, errorList, workingDirectory);
+        result.setFileVersion(ingestionFlowFile.getFileVersion());
```

Comment:
this is wrong, just the import of DP_INSTALLMENTS knows during import the version, here you should determine it based on the profile configured during the read.
Actually, let's set it to 1.0
```suggestion
        result.setFileVersion("1.0");
```